client/app/account/user/agreements/details/user-agreements-details.html

```diff
@@ -60,17 +56,17 @@ <h2 data-translate="user_agreement_details_subtitle3"></h2>
 
         <div data-ng-if="!ctrl.loading">
             <div class="row">
-                <div class="col-xs-12 col-md-6">
+                <div class="col-xs-12 col-md-12">
```

Comment:
Using a row with `col-xs-12 col-md-12` is a non-sense. Simply use a unique `div`.